File: app/src/main/java/ru/henridellal/dialer/OnCallLogScrollListener.java
package ru.henridellal.dialer;

import android.widget.AbsListView;
import android.widget.AbsListView.OnScrollListener;

public class OnCallLogScrollListener implements OnScrollListener {

	private DialerActivity activity;

	public OnCallLogScrollListener(DialerActivity activity) {
		this.activity = activity;
	}

	public void onScroll(AbsListView view, int firstVisibleItem, int visibleItemCount, int totalItemCount) {
	
	}

	public void onScrollStateChanged(AbsListView view, int scrollState) {
		if (scrollState != SCROLL_STATE_IDLE && Numpad.isVisible(activity)) {
			Numpad.hide(activity);
		}
	}
}

File: app/src/main/java/ru/henridellal/dialer/OnCallLogScrollListener.kt
package ru.henridellal.dialer

import android.widget.AbsListView

class OnCallLogScrollListener(private val activity: DialerActivity) : AbsListView.OnScrollListener {
    override fun onScroll(view: AbsListView, firstVisibleItem: Int, visibleItemCount: Int, totalItemCount: Int) {}
    override fun onScrollStateChanged(view: AbsListView, scrollState: Int) {
        if (scrollState != AbsListView.OnScrollListener.SCROLL_STATE_IDLE && activity.isNumpadVisible) {
            activity.hideNumpad()
        }
    }
}